hack/.golint_failures

```diff
@@ -801,6 +801,7 @@ staging/src/k8s.io/sample-apiserver/pkg/registry/wardle/flunder
 test/e2e
 test/e2e/apimachinery
 test/e2e/apps
+test/e2e/auth
```

Comment:
I'd like to add here like other pkg, one problem is that golint do not allow dot impots like:
```
. "github.com/onsi/ginkgo"
. "github.com/onsi/gomega"
```